apps/docs/pages/guides/auth/phone-login/twilio.mdx

```diff
@@ -13,21 +13,49 @@ In this guide we'll show you how to authenticate your users with SMS based One-T
 
 There are two reasons to use Supabase SMS OTP tokens:
 
-- You want users to log in with mobile number + password, and the mobile number should be verified via SMS
-- You want users to log in with mobile number ONLY (i.e. passwordless login)
-
-We'll cover:
-
-- [Finding your Twilio credentials](#finding-your-twilio-credentials)
-- [Using OTP with password based logins](#using-otp-with-password-based-logins)
-- [Using OTP as a passwordless sign-in mechanism](#using-otp-as-a-passwordless-sign-in-mechanism)
+- You want users to log in with a phone number and password, and verify the phone number on signup with SMS
+- You want users to log in with a phone number ONLY (i.e. passwordless login)
 
 What you'll need:
 
 - A Twilio account (sign up here: https://www.twilio.com/try-twilio)
 - A Supabase project (create one here: https://app.supabase.com)
 - A mobile phone capable of receiving SMS
 
+SMS Authentication can be done with either Twilio (Programmable Messaging) or Twilio Verify. [Twilio Verify](https://www.twilio.com/en-us/trusted-activation/verify) is a service that replaces [Twilio Programmable Messaging](https://www.twilio.com/docs/messaging). Twilio Verify is a specialized OTP solution and is recommended for developers requiring features like fraud protection.
```

Comment:
```suggestion
SMS Authentication can be done with either Twilio Verify or Twilio Programmable Messaging. [Twilio Verify](https://www.twilio.com/en-us/trusted-activation/verify) is a specialized OTP solution and is recommended for most developers that need over-the-phone authentication. Alternatively you can use [Twilio Programmable Messaging](https://www.twilio.com/docs/messaging) which offers generic SMS sending support.
```

I want us to focus people on Twilio Verify, as Programmable Messaging is not an authentication solution and there is regulation increasing all over the world about just sending SMSs. All of this is taken care of with Twilio Verify already.

Reply:
this is much better.

Reply:
when I first read through this it wasn't really clear that 1 was better / easier.

I really had to think reading it. don't make me think.

even go as far as put in place callouts explaining in programmable option is not what we recommend because X,Y,Z.